checks/load.go

```diff
@@ -21,6 +21,10 @@ var availableChecks = map[string]ArgableFunc{
 		args["interpreter"] = "/bin/bash"
 		return RunScript{}.FromArgs(args)
 	},
+	"run-python-script": func(args map[string]interface{}) (Checker, error) {
+		args["interpreter"] = "python"
+		return RunScript{}.FromArgs(args)
+	},
```

Comment:
Can we add specific ones for like `run-python2-script` and `run-python3-script`?

Reply:
Added